submissions/VBystrov/html-movie-seat-booking/css/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Noto Sans', sans-serif;
+  font-size: 16px;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+}
+
+.cinema {
+  width: 900px;
+  background-color: #e0e0d1;
+  margin: 20px;
+  box-shadow: 0 0 5px 5px grey;
+  border-radius: 25px;
+}
+
+.header {
+  text-align: center;
+  margin: 20px;
+}
+
+.screen {
+  text-align: center;
+  font-size: 26px;
+  margin-bottom: -20px;
+}
+
+.seats {
+  display: grid;
+  grid-auto-rows: 50px;
+  grid-template-columns: 1fr 8fr;
+  font-size: 26px;
+  gap: 10px;
+  align-items: center;
+  margin-top: 25px;
+  margin-right: 10%;
+}
+
+.row-count {
+  justify-self: end;
+  margin: 0 10px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.seat-place {
```

Comment:
Add `cursor: pointer` for better identification interactive elements.

Reply:
Done.

Reply:
No need to add `cursor: pointer` for info seats.

![image](https://user-images.githubusercontent.com/16196199/107196839-cfdce480-69fb-11eb-9595-5f086571da11.png)

Reply:
Added style for info seats with default cursor.